lib/bolt/cli.rb

```diff
@@ -858,77 +873,57 @@ def initialize_project
 
     # Installs modules declared in the project configuration file.
     #
-    def install_project_modules
-      if config.project.modules.nil?
-        outputter.print_message "Project configuration file '#{config.project.project_file}' "\
-                                "does not specify any module dependencies. Nothing to do."
+    def install_project_modules(project, force)
+      require 'bolt/module_installer'
```

Comment:
If this is expensive to load it seems easier to move requires into the actions in the required files (like in the puppetfile/installer) than to have requires all over the place here.